src/pages/checkout/index.tsx

```diff
@@ -16,47 +18,78 @@ type Address = {
 const Checkout: React.FC = () => {
 	const { isLoggedIn, user } = useAuth();
 	const { loadCartData, cartData } = useCart();
+	const { handleOpenPopup, handleClosePopup } = usePopup();
 	const location = useLocation();
-	const recievedType = location.state?.orderType;
+	const receivedType = location.state?.orderType;
+	const deliveryType = receivedType ? receivedType : 'Point of delivery';
```

Comment:
Более короткая запись:

```suggestion
	const deliveryType = receivedType ||'Point of delivery';
```

А вообще все строковые значения лучше вынести в константы или enum если можно сгруппировать)

Reply:
добавил enum для типов доставки и оплаты

Reply:
Исправлено